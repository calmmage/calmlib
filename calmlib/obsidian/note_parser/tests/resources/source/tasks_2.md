- [ ] 01
    - unchecked task item
- [x] 02
    - checked task item
- 03
    - regular task item
- [ ] 04
    - text
- 05
    - [ ] task text
- [x] 06
    - [ ] text
