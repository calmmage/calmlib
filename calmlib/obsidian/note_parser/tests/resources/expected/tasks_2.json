{
  "items": [
    {
      "type": "task",
      "text": "01",
      "checked": false,
      "children": [
        {
          "type": "list_item",
          "text": "unchecked task item"
        }
      ]
    },
    {
      "type": "task",
      "text": "02",
      "checked": true,
      "children": [
        {
          "type": "list_item",
          "text": "checked task item"
        }
      ]
    },
    {
      "type": "task",
      "text": "03",
      "children": [
        {
          "type": "list_item",
          "text": "regular task item"
        }
      ]
    },
    {
      "type": "task",
      "text": "04",
      "checked": false,
      "children": [
        {
          "type": "list_item",
          "text": "text"
        }
      ]
    },
    {
      "type": "task",
      "text": "05",
      "children": [
        {
          "type": "list_item",
          "text": "task text",
          "checked": false
        }
      ]
    },
    {
      "type": "task",
      "text": "06",
      "checked": true,
      "children": [
        {
          "type": "list_item",
          "text": "text",
          "checked": false
        }
      ]
    }
  ]
}